src/collections/components/CollectionForm/CollectionForm.tsx

```diff
@@ -141,10 +142,8 @@ export const CollectionForm: React.FC<
     validateOnChange: false,
     validationSchema: getValidationSchema(authorIds),
     onSubmit: (values, formikHelpers) => {
-      // TODO: follow up with a separate PR/ticket that will be solely
-      // responsible for validating the labels before submission
-      // as they don't go through our normal form validation
-      onSubmit(values, formikHelpers, selectedLabels);
+      validateLabels(values.labels, selectedLabels) &&
+        onSubmit(values, formikHelpers, selectedLabels);
```

Comment:
You would need to pass the `labels` variable here rather than `values.labels` - the `labels` value is the list of all available labels passed to the form component and destructured out of the props [here](https://github.com/Pocket/curation-admin-tools/blob/60824313fb470a7328ca15f1387bfec9f885353a/src/collections/components/CollectionForm/CollectionForm.tsx#L89).